src/data.yaml

```diff
@@ -2934,6 +2934,19 @@ jobs:
         long: 
     review: 
     rating: 
+  - name: Advanced Infrastructure
+    website: https://www.advanced-infrastructure.co.uk/
+    jobs: https://www.advanced-infrastructure.co.uk/career
+    description: Our mission is to "accelerate an informed, fair and affordable transition to a net zero energy system"
+    remote: false
+    speculative: false
+    field: Misc.
+    geo:
+      - country: UK
+        lat: 
```

Comment:
Could you add the coordinates for Cambridge?

Also, please add Greece and India without coordinates

> We were founded and remain headquartered in Cambridge, England, although today as a remote first business, our team are found right across the world, including Greece, India and the UK.